src/dodal/devices/i15/dcm.py

```diff
@@ -0,0 +1,77 @@
+from typing import Generic, TypeVar
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motor import Motor
+
+from dodal.devices.common_dcm import (
+    StationaryCrystal,
+)
+
+
+class ThetaYCrystal(StationaryCrystal):
+    def __init__(self, prefix):
+        with self.add_children_as_readables():
+            self.theta = Motor(prefix + "THETA")
+            self.y = Motor(prefix + "Y")
+        super().__init__(prefix)
+
+
+class ThetaRollYZCrystal(ThetaYCrystal):
+    def __init__(self, prefix):
+        with self.add_children_as_readables():
+            self.roll = Motor(prefix + "ROLL")
+            self.z = Motor(prefix + "Z")
+        super().__init__(prefix)
+
+
+Xtal_1 = TypeVar("Xtal_1", bound=StationaryCrystal)
+Xtal_2 = TypeVar("Xtal_2", bound=StationaryCrystal)
+
+
+class DualCrystalMonoSimple(StandardReadable, Generic[Xtal_1, Xtal_2]):
+    """
+    Device for simple double crystal monochromators (DCM), which only allow energy of the beam to be selected.
+
+    Features common across all DCM's should include virtual motors to set energy/wavelength and contain two crystals,
+    each of which can be movable. Some DCM's contain crystals with roll motors, and some contain crystals with roll and pitch motors.
+    This base device accounts for all combinations of this.
+
+    This device is more able to act as a parent for beamline-specific DCM's, in which any other missing signals can be added,
+    as it doesn't assume WAVELENGTH, BRAGG and OFFSET are available for all DCM deivces, as BaseDCM does.
+
+    Bluesky plans using DCM's should be typed to specify which types of crystals are required. For example, a plan
+    which only requires one crystal which can roll should be typed 'def my_plan(dcm: BaseDCM[RollCrystal, StationaryCrystal])`
```

Comment:
Should: If `DualCrystalMonoSimple` is meant to be a common DCM then why is it in i15 devices and why is it not used on other beamlines? This may not be in the scope of this issue but if it's not we need issues for tracking the work of combining these and ideally someone to take ownership of it